src/sentry/api/endpoints/organization_events.py

```diff
@@ -369,6 +371,7 @@ def _data_fn(scopedDataset, offset, limit, query) -> dict[str, Any]:
                 use_metrics_layer=batch_features.get("organizations:use-metrics-layer", False),
                 on_demand_metrics_enabled=on_demand_metrics_enabled,
                 on_demand_metrics_type=on_demand_metrics_type,
+                fallback_to_transactions=fallback_to_transactions,
```

Comment:
Could this be a feature flag check within the `metrics_enhanced_performance`'s `.query` function? This parameter does not apply to any other dataset but is being added to every one.

Reply:
I spoke to Shruthi about this and we wanted to control the rollout strictly so we can handle dashboards without affecting the performance overview page. I agree that adding the parameter to other datasets is pretty ugly but we do the same with other params currently and this one is intended to be temporary